CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+We welcome contributions to our code, and will always endeavor to respond in a quick and helpful manner to Pull Requests.
+
+1. **Please sign one of the contributor license agreements below** (simple and pain-free).
+1. Fork the repo, develop and test your code changes, add docs.
+1. Make sure that your commit messages clearly describe the changes.
+1. Send a pull request.
+
+## Contributor License Agreements
+
+Before we can accept your pull requests you'll need to sign a Contributor
+License Agreement (CLA):
+
+- **If you are an individual writing original source code** and **you own the intellectual property**,
+then you'll need to sign an [individual CLA][individual-cla].
+- **If you work for a company that wants to allow you to contribute your work**,
+then you'll need to sign a [corporate CLA][corporate-cla].
+
+You can sign these electronically (just scroll to the bottom). After that,
+we'll be able to accept your pull requests.
+
+[individual-cla]: https://developers.google.com/open-source/cla/individual
+[corporate-cla]: https://developers.google.com/open-source/cla/corporate
+
+# Code Layout
+
+These libraries are split into:
+
+- [Generated client libraries][client-libs]: The per-service libraries as used by users. The source code for these is generated by the Client Generator
+- [Client Generator][client-gen]: The code generator for the above.
+- [Support Libraries][support-libs]: Support code for the client libraries.
+
+[support-libs]: https://github.com/google/google-api-dotnet-client/tree/master/Src/Support
+[client-libs]: https://github.com/google/google-api-dotnet-client/tree/master/Src/Generated
+[client-gen]: https://github.com/google/google-api-dotnet-client/tree/master/ClientGenerator
+
+## Building and .NET Core
+
+We use Visual Studio 2015 to edit the Support Libraries.
+
+[MSBuild][msbuild] is currently used to run the Client Generator, and to build both the generated Client Libraries and the Support Libraries.
+
+We are gradually transitioning towards using [dotnet cli][dotnetcli] for testing and building (see [#822][822]). Until this is complete there are both `csproj` and `project.json` projects for all support libraries.
+
+[msbuild]: https://msdn.microsoft.com/en-us/library/0k6kkbsd.aspx
+[dotnetcli]: https://github.com/dotnet/cli
+[822]: https://github.com/google/google-api-dotnet-client/issues/822
+
+## Tests
+
+Support Library tests use [NUnit][nunit].
+
+Tests for the `.NET45` platform are run using [run`_`tests.bat][runtests], or can be run directly from within Visual Studio.
```

Comment:
Any reason the backticks aren't around the whole of this, and in the line below?